deno/payloads/v10/_interactions/applicationCommands.ts

```diff
@@ -74,6 +74,8 @@ export interface APIApplicationCommand {
 	default_member_permissions: Permissions | null;
 	/**
 	 * Indicates whether the command is available in DMs with the app, only for globally-scoped commands. By default, commands are visible
+	 *
+	 * @deprecated Use `contexts` instead
 	 */
 	dm_permission?: boolean;
```

Comment:
I don't think this shoud be deprecated rn. Because contexts property is unstable.
```suggestion
	// TODO: deprecate after APIApplicationCommand.contexts become stable
	/**
	 * Indicates whether the command is available in DMs with the app, only for globally-scoped commands. By default, commands are visible
	 */
	dm_permission?: boolean;
```

Reply:
If this is to reflect Discord pedantically, perhaps it should be deprecated.

Deprecating a property for a field in beta seems strange too, though.

Reply:
It's odd but I did it to reflect exactly what the API docs say https://discord.com/developers/docs/interactions/application-commands#application-command-object

Depends how strictly we want to stay with the docs.

Reply:
I don't wanna block this PR any longer, lets stick to what the docs say. If they undeprecate dm_permission, so will we. Its not the end of the world :D